typescript/src/schema/proto.ts

```diff
@@ -162,7 +167,18 @@ function toLRInterface(type: string, inputType: string) {
   return inputType.replace(/\.([^.]+)$/, '.I' + type);
 }
 
-function augmentMethod(messages: MessagesMap, method: MethodDescriptorProto) {
+function getComments(
+  method: MethodDescriptorProto,
+  comments: CommentsMap
+): string {
+  return comments[method.name!].trim();
```

Comment:
minor suggestion: let's make `CommentsMap` a class and this `getComments` function will become its method. `getCommentsMap` will then become a constructor. It can go to a separate file `comments.ts`.

Reply:
Will do!